src/snowflake/snowpark/modin/plugin/docstrings/series.py

```diff
@@ -1833,7 +1835,7 @@ def map():
 
         ``map`` accepts a ``dict`` or a ``Series``. Values that are not found
         in the ``dict`` are converted to ``NaN``, unless the dict has a default
-        value (e.g. ``defaultdict``):
+        value (e.g. ``defaultdict``) (Currently not supported by Snowpark pandas):
```

Comment:
the reason we want to keep those unsupported things in the doc is we don't want to forget about thos?

Reply:
That's correct.